config/deploy/manifests.yaml

```diff
@@ -904,7 +904,7 @@ spec:
                               description: OAuth2 Client ID.
                               type: string
                             clientSecretRef:
-                              description: Reference to a Kuberentes Secret key that
+                              description: Reference to a Kubernetes Secret key that
```

Comment:
This actually doesn't belong to this PR. I've sent a separate one: https://github.com/Kuadrant/authorino-operator/pull/127